ts/pages/account/activity.tsx

```diff
@@ -60,6 +62,9 @@ export const AccountActivity: React.FC<ActivityProps> = () => {
     const networkId = useSelector((state: State) => state.networkId);
     const dispatch = useDispatch();
 
+    const dimensions = useWindowDimensions();
```

Comment:
I'm completely fine with your solution but there is also a screen width metric available in the state.
`const screenWidth = useSelector((state: State) => state.screenWidth);`

`screenWidth === 40` would be your mobile width check.

Reply:
thanks for the heads up! I'll change it. No good to introduce new code unnecessarily

Reply:
I decided to keep this as is, because this dynamically adjusts to screen size as well. The other method appears to be static on page load.